napari/components/dims.py

```diff
@@ -33,6 +33,8 @@ class Dims:
         Flag if to clip indices based on range. Needed for image-like
         layers, but prevents shape-like layers from adding new shapes
         outside their range.
+    axis_labels : list of str
```

Comment:
Can you also add `axis_labels=None` as a keyword argument to the Dims, and also `ndisplay=2`, and `order=None`, so the function signature is now
```python
def __init__(self, init_ndim=0, ndisplay=2, order=None, axis_labels=None)
```
and the initialize those variables as appropriate if `None` is passed, or use the passed values otherwise. Can you also add a check the `len(order)` and `len(axis_labels)` is equal to `init_ndim` if neither of them are `None`